Resources/Locale/en-US/components/kitchen-spike-component.ftl

```diff
@@ -1,6 +1,6 @@
 comp-kitchen-spike-deny-collect = { THE($this) } already has something on it, finish collecting its meat first!
 comp-kitchen-spike-deny-butcher = { THE($victim) } can't be butchered on { THE($this) }.
-comp-kitchen-spike-deny-not-dead = { THE($victim) } can't be butchered. { THE($victim) } is not dead!.
+comp-kitchen-spike-deny-not-dead = { THE($victim) } can't be butchered. { THE($victim) } is not dead!
```

Comment:
Can probably change the second sentence to be `{ CAPITALIZE(SUBJECT($victim)) } { CONJUGATE-BE($victim) }`

Need most of these to be capitalized too.

Reply:
conjugate-be doesn't make sense there because it's using their name, you'd get stuff like `The monkey are not dead!`, in this case it would switch on plurality and i don't see why the subject here would ever be plural

Reply:
I was going for:
John Doe can't be butchered. He is not dead.
The monkey can't be butchered. It is not dead."
However that works out.

Reply:
oh nvm i see what you mean, i'll do that